apps/dashboard/src/contract-ui/tabs/nfts/components/claim-button.tsx

```diff
@@ -40,7 +24,7 @@ export const NFTClaimButton: React.FC<NFTClaimButtonProps> = ({
         onClose={onClose}
         isOpen={isOpen}
       >
-        <NFTClaimForm contract={contractQuery.contract} />
+        <NFTClaimForm contractAddress={contractAddress} chainId={chainId} />
```

Comment:
I wonder if this implicitly means we are removing support for some older claim types? does v5 claimTo extension handle all the v4 supported cases? I legitimately do not know cc @joaquim-verges @gregfromstl?

Reply:
if it does mean that then same as with marketplace that's probably OK we just have to make sure support is informed of this - so we're not surprised of _possible_ issues

Reply:
Will imform the guys about this change!